judge/jinja2/datetime.py

```diff
@@ -26,7 +27,7 @@ def wrapper(datetime, *args, **kwargs):
 
 @registry.function
 def relative_time(time, **kwargs):
-    abs_time = date(time, kwargs.get('format', _('N j, Y, g:i a')))
+    abs_time = date(time.astimezone(timezone.get_current_timezone()), kwargs.get('format', _('N j, Y, g:i a')))
```

Comment:
Does this work when the user is logged out? Ideally, we should use the browser's timezone when they are logged out.

Reply:
I think that might have to be fixed separately in timezone.py